src/container/BlockDetailPage/BlockDetailPage.tsx

```diff
@@ -29,18 +29,21 @@ const BlockDetailPage = () => {
   const { id } = useParams<ParamTypes>()
 
   const blockIdentifier = {} as { hash?: string; sequence?: number }
+  const queryParams: any = {}
+
   if (Number.isNaN(Number(id))) {
     blockIdentifier.hash = id
+    queryParams.hash = id
+    queryParams.with_transactions = true
   } else {
     blockIdentifier.sequence = Number(id)
+    queryParams.sequence = Number(id)
+    queryParams.with_transactions = true
   }
```

Comment:
Since `with_transactions` is getting set regardless, would this work?
```suggestion
  const queryParams: any = { with_transactions: true }

  if (Number.isNaN(Number(id))) {
    blockIdentifier.hash = id
    queryParams.hash = id
  } else {
    blockIdentifier.sequence = Number(id)
    queryParams.sequence = Number(id)
  }
```